doc/content/gateways/mikrotik-knot/_index.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "MikroTik KNOT"
+description: ""
+weight: 1
+---
+
+{{< figure src="mikrotik-knot.jpeg" alt="Ltmikrotik-knotAP" class="plain float" >}}
+
+The MikroTik KNOT LR8/LR9 kit is a universal device with exceptional connectivity options and protocol support. It is an IoT Gateway which can use LTE-M (also known as CAT-M) technology as the gateway backhaul. 
+
+<!--more-->
+
+For more info, see Mikrotik's [official product page](https://mikrotik.com/product/knot)
+
+{{< warning >}} Currently, this gateway does not support changing Frequency Plans. {{</ warning >}}
+
+## Prerequisites
+
+1. User account on {{% tts %}} with rights to create Gateways.
+2. MikroTik KNOT connected via Ethernet or LTE-M.
+3. A web browser (see section [Configuration via Browser](#configuration-via-browser)), or the [MikroTik Mobile App](https://mikrotik.com/mobile_app) installed on a smartphone.
+
+## Registration
+
+Create a gateway by following the instructions for [Adding Gateways]({{< ref "/gateways/adding-gateways" >}}). The **EUI** of the gateway can be found on the back panel of the gateway under the field **GW ID**, or under the **Gateway ID** in the LoRa tab of the MikroTik web GUI.
+
+## Configuration via Browser
+
+The MikroTik Routerboard exposes a WiFi Access Point (AP) with SSID `MikroTik-xxxxxx`, where `xxxxxx` are the last 6 digits of the device's MAC Address.
+
+Use your PC, tablet or smartphone to connect to this AP. You will not need a password to connect to this AP.
+
+Open a browser to `http://192.168.88.1/webfig/#Interfaces`.
+
+![MikroTik-portal.png](portal.png)
+
+Select **LoRa** on the left hand menu. The status of the gateway should be **Disabled** - if needed, press the **D** button to disable it.
+
+![MikroTik-portal-lora.png](portal-lora.png)
+
+{{< note >}}
+The Gateway ID you see here is the Gateway EUI you need to register the gateway in The Things Stack.
+{{</ note >}}
+
+From the top menu, select the **Servers** Tab and press the **Add New** button.
+
+![MikroTik-server](servers.png)
+
+Edit the server parameters.
+
+- **Name**: A distinct name
+- **Address**: Address of the Gateway Server. If you followed the [Getting Started guide]({{< ref "/getting-started" >}}) this is the same as what you use instead of `thethings.example.com`.
```

Comment:
```suggestion
- **Address**: Address of {{% tts %}} Gateway Server. If you followed the [Getting Started guide]({{< ref "/getting-started" >}}) this is the same as what you use instead of `thethings.example.com`.
```

Reply:
What does one use if he didn't follow the getting started guide? For example if he's using TTS Community Edition?